### ISSUE TITLE
Accessibility Bug - .modal-layout background overlaps text

### ISSUE THREAD

**Issue by edent** [mastodon/mastodon#18080]
### Steps to reproduce the problem

1. Attempt to interact with someone on a different server.
2. A modal window pops up.
3. The text at the bottom is difficult to read.

![Text at the bottom is hard to read due to a graphic.](https://user-images.githubusercontent.com/837136/165130548-806b3d06-4f46-4605-b30b-ff896905e3c3.png)
![The same bug in a different modal.](https://user-images.githubusercontent.com/837136/165130545-e2dc2f51-98cf-430d-9d60-bbc4de42c179.png)

### Expected behaviour

Text should be high enough contrast to meet relevant WCAG guidance.

### Actual behaviour

Text is difficult to read

### Specifications

See on multiple Mastodon instances, using Firefox and Chrome.

The issue is caused by the CSS which sets the `background` SVG for `<body class="modal-layout"`

**Comment by edent** [comment]
The specific line of CSS is at https://github.com/mastodon/mastodon/blob/f47a9ddc9ffca22258ec9e4b12ca51db8cac1eac/app/javascript/styles/mastodon/modal.scss#L2

I'd suggest either removing it or reducing the opacity.  The current colour is `#606984`, changing it to `#60698433` would make the text easier to read.

**Comment by mgifford** [comment]
Would love to see this change merged in.

**Comment by Gargron** [comment]
This modal is no longer in use.

**Comment by mgifford** [comment]
Ok, guess it can be closed then....
